### ISSUE TITLE
"Slow disk" test cases

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#1651]
Recent issues revealed xud hitting edge cases/crashed when running on a slow disk, delaying trade execution. We should have standardized swap tests with such a setup for releases.

Once you wrote the test cases and executed them once, we can close here.

**Comment by raladev** [comment]
> Recent issues revealed xud hitting edge cases/crashed when running on a slow disk, delaying trade execution. We should have standardized swap tests with such a setup for releases.
> 
> Once you wrote the test cases and executed them once, we can close here.

Any performance case should be based on requirements.
Do we have sort of requirements except of 'use ssd instead of hdd' suggestion?
If we have no these requirements, then i suggest to find them for L1 and L2 and use them as baseline with 1.2/1.5/2/x multiplier and any problem that appear with a speed below that should not be solved by any fixes in xud.
WDYC @kilrau ?

**Comment by kilrau** [comment]
Good point. I would set our goal for a single xud-docker full (without geth) environment to function and swap properly on an (unused) disk with a real-world USB 2.0 speed of 30 MB/s.
